README.md

```diff
@@ -43,24 +43,26 @@ First of all, install `rustc` and `cargo` following the [official instructions](
 ### 1. Dictionary preparation
 
 You can easily get started with Vibrato by downloading a precompiled dictionary.
+[The Releases page](https://github.com/daac-tools/vibrato/releases) distributes
+several precompiled dictionaries from different resources.
+
 Here, consider to use [mecab-ipadic v2.7.0](https://taku910.github.io/mecab/).
 
-```
-$ wget https://github.com/daac-tools/vibrato/releases/download/v0.3.1/ipadic-mecab-2_7_0.tar.gz
+```shell
+# Specify an appropriate release tag to VERSION (e.g., v0.4.0).
+$ wget https://github.com/daac-tools/vibrato/releases/download/VERSION/ipadic-mecab-2_7_0.tar.gz
 $ tar -xzf ipadic-mecab-2_7_0.tar.gz
```

Comment:
Could you use `.tar.xz`?

Reply:
Sure. https://github.com/daac-tools/vibrato/pull/112/commits/1dba2931858257e83e322bdedfe0c1f636d726c4